### ISSUE TITLE
Always no Thumbnails & photos after upgrading to 5.3.1~ynh1

### ISSUE THREAD

**Issue by kazukyakayashi** [YunoHost-Apps/lychee_ynh#115]
### Describe the bug

The problem with thumbnails and photos is still there, obviously a problem of access and writing rights (?).

### Context

- Hardware: OVH (Kimsufi) KS-5 - Intel Xeon E5-1620v2 - 4c/8t - 3.7 GHz/3.9 GHz, 32go RAM
- YunoHost version: 1.2.14.1 (stable)
- If upgrading, current package version: upgrading to 5.3.1~ynh1

### Logs

```
    Diagnostics
    -----------
        Warning: /ho****************************ds/ has permissions 2755, but should have 2775
        Error: /ho****************************ds/ is not writable by lychee
        Warning: /ho****************************ds/big has permissions 0750, but should have 2775
        Warning: /ho****************************ds/raw has permissions 0750, but should have 2775
        Warning: /ho****************************ds/thumb2x has permissions 2755, but should have 2775
        Warning: /ho****************************ds/thumb2x/50 has permissions 2755, but should have 2775
        Warning: /v******************ic/sym/ is owned by group www-data, but should be owned by one out of lychee
        Warning: /var**************orage/image-jobs/ is owned by group www-data, but should be owned by one out of lychee
        Warning: /var**************orage/livewire-tmp/ is owned by group www-data, but should be owned by one out of lychee
        Warning: 1745 more directories with wrong permissions
        Warning: Dropbox import not working. dropbox_key is empty.
        Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
        Info: Latest version of PHP is 8.3
        Warning: Default timezone not properly set; you might experience strange results when importing photos without explicit EXIF timezone
```

**Comment by ericgaspar** [comment]
can you try to apply the right permission to this folder? `chown -R 2775 /home/yunohost.app/lychee/`

**Comment by kazukyakayashi** [comment]
```
ls -l /home/yunohost.app/lychee/
total 4
drwxr-sr-x 12 2775 lychee 4096 Jan  9 19:39 uploads
```

But the problem persists.

```
    Diagnostics
    -----------
        Warning: /ho****************************ds/ has permissions 2755, but should have 2775
        Error: /ho****************************ds/ is not writable by lychee
        Warning: /ho****************************ds/big has permissions 0750, but should have 2775
        Error: /ho****************************ds/big is not writable by lychee
        Warning: /ho****************************ds/raw has permissions 0750, but should have 2775
        Error: /ho****************************ds/raw is not writable by lychee
        Warning: /ho****************************ds/thumb2x has permissions 2755, but should have 2775
        Error: /ho****************************ds/thumb2x is not writable by lychee
        Warning: /ho****************************ds/thumb2x/50 has permissions 2755, but should have 2775
        Error: /ho****************************ds/thumb2x/50 is not writable by lychee
        Warning: /v******************ic/sym/ is owned by group www-data, but should be owned by one out of lychee
        Warning: /var**************orage/image-jobs/ is owned by group www-data, but should be owned by one out of lychee
        Warning: /var**************orage/livewire-tmp/ is owned by group www-data, but should be owned by one out of lychee
        Warning: 1745 more directories with wrong permissions
        Warning: 1730 more inaccessible directories
        Warning: Dropbox import not working. dropbox_key is empty.
        Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
        Info: Latest version of PHP is 8.3
        Warning: Default timezone not properly set; you might experience strange results when importing photos without explicit EXIF timezone
```

**Comment by Ni-Ti** [comment]
I tried to update to version 5.4.0 (testing), the app's diagnosis after updating: 

```
    Diagnostics
    -----------
        Warning: /ho****************************ds/ is owned by group www-data, but should be owned by one out of lychee
        Error: /ho****************************ds/ is not writable by lychee
        Warning: /ho****************************ds/thumb2x is owned by group www-data, but should be owned by one out of lychee
        Error: /ho****************************ds/thumb2x is not writable by lychee
        Warning: /ho****************************ds/thumb2x/51 is owned by group www-data, but should be owned by one out of lychee
        Error: /ho****************************ds/thumb2x/51 is not writable by lychee
        Warning: /ho****************************ds/thumb2x/51/c4 is owned by group www-data, but should be owned by one out of lychee
        Error: /ho****************************ds/thumb2x/51/c4 is not writable by lychee
        Warning: /ho****************************ds/thumb2x/51/0f is owned by group www-data, but should be owned by one out of lychee
        Error: /ho****************************ds/thumb2x/51/0f is not writable by lychee
        Warning: /ho****************************ds/medium has permissions 0750, but should have 2775
        Warning: /ho****************************ds/import has permissions 0750, but should have 2775
        Warning: /ho****************************ds/big has permissions 0750, but should have 2775
        Warning: /ho****************************ds/raw has permissions 0750, but should have 2775
        Warning: /ho****************************ds/small has permissions 0750, but should have 2775
        Warning: 15171 more directories with wrong owner
        Warning: 4 more directories with wrong permissions
        Warning: 15174 more inaccessible directories
        Warning: Dropbox import not working. dropbox_key is empty.
        Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
        Warning: Default timezone not properly set; you might experience strange results when importing photos without explicit EXIF timezone
    

  System Information
    ------------------
        Lychee Version (release):                5.4.0
        DB Version:                              5.4.0
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 set
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.3.8
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                Etc/UTC
        Max uploaded file size:                  1G
        Max post size:                           1G
        Livewire chunk size:                     12.00 MB
        Max execution time:                      30
        MySQL Version:                           10.5.23-MariaDB-0+deb11u1
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1691
        GD Version:                              2.3.3
        Number of foreign key:                   12 found.
```

About folder permissions:

```
/home/yunohost.app/lychee/uploads# ll
total 44
drwxr-x---   2 2775 lychee   4096 Jan 27 15:31 big
drwxr-x---   2 2775 lychee   4096 Jan 27 15:31 import
drwxrwsr-x   2 2775 www-data 4096 Apr 18 19:28 livewire-tmp
drwxr-x--- 258 2775 lychee   4096 Jan 27 18:16 medium
drwxrwsr-x 258 2775 www-data 4096 Mar 10 17:50 medium2x
drwxrwsr-x 258 2775 www-data 4096 Jan 27 18:16 original
drwxr-x---   2 2775 lychee   4096 Jan 27 15:31 raw
drwxr-x--- 258 2775 lychee   4096 Jan 27 18:16 small
drwxrwsr-x 258 2775 www-data 4096 Jan 27 18:16 small2x
drwxr-x--- 258 2775 lychee   4096 Jan 27 18:16 thumb
drwxrwsr-x 258 2775 www-data 4096 Jan 27 18:16 thumb2x

```

**Comment by ericgaspar** [comment]
we should have the warning removed with the testing branch. still I cannot get the thumbnails displayed :/

**Comment by ericgaspar** [comment]
Thumbnails should be fixed now.

**Comment by Ni-Ti** [comment]
Thumbnails are back. Thank you very much. 🙏

**Comment by ericgaspar** [comment]
fixed with #117
